setup.py

```diff
@@ -18,9 +18,9 @@
     author_email='Richard.Beare@med.monash.edu.au',
     packages=['itk'],
     package_dir={'itk': 'itk'},
-    download_url=r'https://github.com/InsightSoftwareConsortium/LabelErodeDilate',
+    download_url=r'https://github.com/InsightSoftwareConsortium/ITKLabelErodeDilate',
     description=r'An ITK module for erode and dilate operations on label images',
-    long_description='LabelErodeDilate provides classes for morphological math'
+    long_description='ITKLabelErodeDilate provides classes for morphological math'
```

Comment:
Shouldn't this be LabelErodeDilate without ITK prefix?

Reply:
The project name is appropriate here, and the *ITK* prefix helps make the connection with ITK.